biz.aQute.bndlib.comm.tests/test/aQute/bnd/comm/tests/IndexedReposWithComms.java

```diff
@@ -155,6 +156,7 @@ private void createSecureSocks5() throws IOException, InterruptedException {
 		UserManager userManager = new MemoryBasedUserManager();
 		userManager.create(new User("proxyuser", "good"));
 		SocksServerBuilder builder = SocksServerBuilder.newSocks5ServerBuilder();
+		builder.setBindAddr(InetAddress.getByName("localhost"));
```

Comment:
Sorry, `getLoopbackAddress` is the right method to use. We use it in other places.